tensorbay/opendataset/PlantSeedlings/loader.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+# pylint: disable=invalid-name
+# pylint: disable=missing-module-docstring
+
+import os
+from typing import Dict, Iterator, Optional
+
+import numpy as np
+
+from ...dataset import Data, Dataset, Segment
+from ...label import Classification, SemanticMask
+from .._utility import glob
+
+try:
+    from PIL import Image
+except ModuleNotFoundError:
+    from .._utility.mocker import Image
+
+_FULLIMAGES_MAP = {
+    (1, 4): "Maize",
+    (5, 8): "Common wheat",
+    (9, 12): "Sugar beet",
+    (13, 16): "scentless mayweed",
+    (17, 20): "Common Chickweed",
+    (21, 24): "shepherd's-purse",
+    (25, 28): "Cleavers",
+    (29, 32): "Fersken pileurt",
+    (33, 36): "Charlock",
+    (37, 40): "Fat Hen",
+    (41, 44): "Cranes-bill",
+    (45, 48): "Field Pansy",
+    (49, 52): "Black grass",
+    (53, 56): "Loose Silky-bent",
+}
+
+_CATEGORY_ID_MAP = {
```

Comment:
can we get this map from `catalog`?

Reply:
Their names have a little bit difference.